client/my-sites/themes/theme-showcase.jsx

```diff
@@ -287,7 +288,9 @@ class ThemeShowcase extends Component {
 			this.setState( { tabFilter: this.tabFilters.ALL } );
 		}
 
+		recordTracksEvent( 'calypso_themeshowcase_filter_pricing_click', { tier } );
```

Comment:
Should this be behind feature flag `themes/showcase-i4/search-and-filter`?

Reply:
I'm fine with not putting Tracks events behind feature flags. They are cheap.